securethenews/blog/models.py

```diff
@@ -56,6 +56,6 @@ def posts(self):
         posts = BlogPost.objects.live().descendant_of(self)
 
         # Order by most recent date first
-        posts = posts.order_by('-date')
+        posts = posts.order_by('-id')
```

Comment:
Ok, so I just tested this out and the "potential problem" that I had originally described does not appear to be a problem after all! I'm not sure why. I guess Wagtail reassigns the primary key when a page is published? Anyway, this fix and test look good after all, so I'm going to ahead and merge this. Sorry for holding up the merge on what turned out to be a non-issue!